### PR TITLE
ContextualMenu: Remove unneeded extra role=menu

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

ContextualMenu: Remove unneeded extra role=menu

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8535)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx [hunk 1]
```diff
@@ -318,7 +318,6 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
           hidden={this.props.hidden}
         >
           <div
-            role="menu"
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx]: run `npm run update-snapshots`

this is going to break a snapshot test.

**Comment by Vitalius1** [comment]
Would be curious what is the output of Narrator before and after this change.

**Comment by joschect** [comment]
@Vitalius1 I'll double check

**Comment by khmakoto** [comment]
@Vitalius1 I just checked and there doesn't appear to be any changes between the live version and this one with Narrator+Keyboard navigation.

**Pull request merged**